lib/any.js

```diff
@@ -683,6 +684,22 @@ module.exports = internals.Any = class {
             }
         }
 
+        if (this._settings) {
+
+            const settings = Hoek.clone(this._settings);
```

Comment:
So all this block can be shortened to `description.options = Hoek.clone(this._settings);` ?